src/main/java/com/netgrif/application/engine/pdf/generator/service/renderer/Renderer.java

```diff
@@ -36,6 +39,9 @@ public void setupRenderer(IPdfDrawer pdfDrawer, PdfResource resource) {
         this.lineHeight = resource.getLineHeight();
         this.pageDrawableWidth = resource.getPageDrawableWidth();
         this.padding = resource.getPadding();
+        this.colorString = Color.decode(resource.getColorString().toUpperCase());
+        this.colorDataGroupLabel = Color.decode(resource.getColorDataGroup());
```

Comment:
All colour strings (hex strings) should be transformed to upper case before decoding.

Reply:
Done